tsconfig.json

```diff
@@ -3,16 +3,18 @@
     "module": "commonjs",
     "moduleResolution": "node",
     "strict": true,
-    "lib": ["ESNext"],
-    "target": "es2018",
+    "lib": ["ESNext", "DOM", "DOM.Iterable"],
+    "target": "es2021",
     "outDir": "lib",
     "declaration": true,
     "resolveJsonModule": true,
     "esModuleInterop": true,
     "isolatedModules": true,
     "noImplicitAny": false,
     "preserveConstEnums": true,
-    "newLine": "lf"
+    "newLine": "lf",
+    "skipLibCheck": true,
```

Comment:
What's this doing?

Reply:
https://www.typescriptlang.org/tsconfig/#skipLibCheck I had some conflicts when I started, seem like it's not needed anymore